contrib/management-packs/isilon-onefs-mpack/src/main/resources/addon-services/ONEFS/1.0.0/quicklinks/quicklinks.json

```diff
@@ -3,26 +3,37 @@
   "description": "default quick links configuration",
   "configuration": {
     "protocol": {
-      "type": "HTTP_ONLY"
+      "type": "HTTPS_ONLY"
     },
     "links": [
       {
         "component_name" : "ONEFS_CLIENT",
         "name": "onefs_web_ui",
         "label": "OneFS Web UI",
-        "url": "%@://%@",
+        "url": "%@://%@:%@",
+        "port": {
+          "http_default_port": "8080",
```

Comment:
Should this (below too) be `80` (or maybe `8082`)?

Reply:
No. `80` points to:
* file browse when http service is `enabled`
* nothing when http service is `disabled`
* webui only when http service is `redirect` (the default value in OneFS; and since not indicating a port on a http query default to 80, that's why this worked most of the time)

`8080` is the safe port.